### ISSUE TITLE
NullConstraintViolation related to DateTime type

### ISSUE THREAD

**Issue by Tyguy160** [prisma/prisma2#315]
Hello,

I am running into some issues in creating a new `User` due to a `DateTime` issue. When I go to create a new User, I receive a `NullConstrainViolation` error:

```Error in Photon: 
ConnectorError(NullConstraintViolation { field_name: "User.createdAt" })
```

This seems to be similar to issue #166, but in this case the DateTime fields shouldn't require a relation since they're auto-populated by Prisma behind the scenes. When I comment out the DateTime fields, there are no issues in creating a new User or Credential. Any help with this would be greatly appreciated!

Here are my models from the `schema.prisma` file:

```
model User {
  id         String     @default(cuid()) @id @unique
  createdAt  DateTime   @default(now())
  updatedAt  DateTime   @updatedAt
  credential Credential
}

model Credential {
  id        String   @default(cuid()) @id @unique
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  email     String   @unique
  password  String
  user      User
}
```

**Comment by divyenduz** [comment]
@Tyguy160 : What mutation do you run exactly? 

Are you trying to connect the user to an existing credential or are you trying to create a credential inline?

**Comment by divyenduz** [comment]
I am able to reproduce this with a nested create: https://github.com/divyenduz/p2_315

For the following mutation: 
```
const user = await photon.users.create({
    data: {
      credential: {
        create: {
          email: 'a@b.com',
          password: 'z',
        },
      },
    },
  })
```

The following error is observed: 

```
(node:48881) UnhandledPromiseRejectionWarning: Error: Error in Photon: 
ConnectorError(NullConstraintViolation { field_name: "User.cjyycy7ij0000pux1xc6jlreb, null, 1970-01-01 00:00:00)." })
    at PhotonFetcher.<anonymous> (/Users/divyendusingh/Documents/prisma/p2-315/node_modules/@generated/photon/index.js:35:23)
    at Generator.throw (<anonymous>)
    at rejected (/Users/divyendusingh/Documents/prisma/p2-315/node_modules/@generated/photon/index.js:5:65)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:48881) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:48881) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by mavilein** [comment]
This is fixed in binary version `b8957df02bca6708b7aefa56832eda409dbb1114`. It is already available on `alpha`.
